Hush/Assets/Plugins/PyroParticles/Prefab/Script/FireCollisionForwardScript.cs

```diff
@@ -1,6 +1,4 @@
 ﻿using UnityEngine;
-using System.Collections;
-using Common;
 
 namespace DigitalRuby.PyroParticles
 {
```

Comment:
Why was this file changed?

Reply:
Remove useless imports + move custom logic to inherited class